thirdparty/github.go

```diff
@@ -32,6 +32,8 @@ const (
 	GithubAPIStatusMajor = "major"
 	GithubAPIStatusGood  = "good"
 
+	Github502Error = "502 Server Error []"
```

Comment:
Should this instead omit the array and check if the error string contains it (in case the array is ever populated)? Also did checking error.Cause() not work for this?

Reply:
I changed the error string to just search for "502 Server Error" and changed to contains.

My thinking for not using error.Cause() was that given this error string:

`API response error: error querying for commits in '...' ref master : GET https://api.github.com/repos/.../.../commits?sha=master: 502 Server Error []`

I wanted to know if the root cause was a 502 Server Error from the http package and I didn't see an advantage for searching for '502 Server Error' in the cause's string (`error querying for commits in '...' ref master : GET https://api.github.com/repos/.../.../commits?sha=master: 502 Server Error []`) vs. the wrapped error.